diffkemp/simpll/Config.h

```diff
@@ -21,6 +21,7 @@
 #include <llvm/Support/SourceMgr.h>
 
 #define DEBUG_SIMPLL "debug-simpll"
+#define VERBOSE_SIMPLL "verbose-simpll"
```

Comment:
`DEBUG_SIMPLL_VERBOSE` would be better here (to be consistent with the other debug types by using the `DEBUG` prefix).

Reply:
What would be the 3rd level called? `DEBUG_SIMPLL_VERBOSE_MACROS` or `DEBUG_SIMPLL_VERBOSE_3`. I don't think either of those would be fitting. The first one says that only macros are at this level, and the third one wouldn't quite fit the naming scheme.

Reply:
We could call it `DEBUG_SIMPLL_VERBOSE_EXTRA`, that way, each level has an added word.